packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -723,17 +799,55 @@ class MemoryBodyState extends State<MemoryBody>
           const PaddedDivider(
             padding: EdgeInsets.only(left: denseRowSpacing),
           ),
-          Image(image: AssetImage(image)),
+          widget,
         ]);
       }
 
       return rightSide;
     }
 
+    Widget legendSymbol(Map<String, Object> dataToDisplay) {
+      final image = dataToDisplay.containsKey(renderImage)
+          ? dataToDisplay[renderImage] as String
+          : null;
+      final color = dataToDisplay.containsKey(renderLine)
+          ? dataToDisplay[renderLine] as Color
+          : null;
+      final dashedLine = dataToDisplay.containsKey(renderDashed)
+          ? dataToDisplay[renderDashed]
+          : false;
+
+      Widget traceColor;
+      if (color != null) {
+        if (dashedLine) {
+          traceColor = Padding(
+            child: CustomPaint(
+              painter: DashedLine(15, color, 2, 4, 3),
+              foregroundPainter: DashedLine(15, color, 2, 4, 3),
+            ),
+            padding: const EdgeInsets.fromLTRB(2, 5, 20, 1),
```

Comment:
these padding consts seem strange. Can we use something a little more consistent?

additionally, child should always be the last param in Widget code, so move `padding` above `child`

Reply:
Don't know why I had such bizarre padding. 

`            padding: const EdgeInsets.only(right: 20),
`